src/modifiers.ts

```diff
@@ -92,190 +93,200 @@ function hasSpecialChars(str: string): boolean {
 
 export class Modifiers {
     codeModel: CodeModel;
-    allCommandGroups: string[];
+    allCommandGroups: Record<string, number>;
 
     constructor(protected session: Session<CodeModel>) {
         this.codeModel = session.model;
         this.getAllCommandGroup();
     }
 
     getAllCommandGroup() {
-        this.allCommandGroups = [];
-        for (const group of values(this.codeModel.operationGroups)) {
-            this.allCommandGroups.push(group.language['az'].command);
-        }
+        this.allCommandGroups = {};
```

Comment:
It looks the operation name  is different from what the operation is doing.

Reply:
changed it into initAllCommandGroup